spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsArgumentParser.scala

```diff
@@ -8,25 +8,17 @@ object SuspiciousConnectsArgumentParser {
 
   case class SuspiciousConnectsConfig(analysis: String = "",
                                       inputPath: String = "",
-                                      scoresFile: String = "",
+                                      feedbackFile: String = "",
                                       duplicationFactor: Int = 1,
-                                      modelFile: String = "",
-                                      topicDocumentFile: String = "",
-                                      topicWordFile: String = "",
-                                      mpiPreparationCmd: String = "",
-                                      mpiCmd: String = "",
-                                      mpiProcessCount: String = "",
                                       topicCount: Int = 20,
-                                      localPath: String = "",
-                                      localUser: String = "",
-                                      ldaPath: String = "",
-                                      nodes: String = "",
                                       hdfsScoredConnect: String = "",
-                                      hdfsModelFile: String = "",
                                       threshold: Double = 1.0d,
                                       maxResults: Int = -1,
                                       outputDelimiter: String = "\t",
-                                      ldaPRGSeed: Option[Long] = None)
+                                      ldaPRGSeed: Option[Long] = None,
```

Comment:
I tried to find where this value is passed but seems like it's being defaulted to -1L in SpotSparkLDAWrapper. The parameter is not available in ml_ops.sh, do we need to keep it?